File: Plugins/Lpp.Dns.General.Metadata/Content/Models/Terms/MetaDataTable.js
/// <reference path="../../../../../Lpp.Mvc.Composition/Lpp.Mvc.Boilerplate/jsBootstrap.d.ts" />
/// <reference path="../Terms.ts" />
var DataCheckerModels;
(function (DataCheckerModels) {
    var MetaDataTableTypes;
    (function (MetaDataTableTypes) {
        MetaDataTableTypes[MetaDataTableTypes["Diagnosis"] = 0] = "Diagnosis";
        MetaDataTableTypes[MetaDataTableTypes["Dispensing"] = 1] = "Dispensing";
        MetaDataTableTypes[MetaDataTableTypes["Encounter"] = 2] = "Encounter";
        MetaDataTableTypes[MetaDataTableTypes["Enrollment"] = 3] = "Enrollment";
        MetaDataTableTypes[MetaDataTableTypes["Procedure"] = 4] = "Procedure";
    })(MetaDataTableTypes = DataCheckerModels.MetaDataTableTypes || (DataCheckerModels.MetaDataTableTypes = {}));
})(DataCheckerModels || (DataCheckerModels = {}));

File: Plugins/Lpp.Dns.General.CriteriaGroupCommon/Models/Terms/MetaDataTable.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;

namespace RequestCriteria.Models
{
    [DataContract]
    public class MetaDataTableData : TermData
    {
        [DataMember]
        public MetaDataTableTypes[] Tables { get; set; }
    }

    [DataContract]
    public enum MetaDataTableTypes
    {
        [EnumMember]
        Diagnosis = 0,
        [EnumMember]
        Dispensing = 1,
        [EnumMember]
        Encounter = 2,
        [EnumMember]
        Enrollment = 3,
        [EnumMember]
        Procedure = 4,
    }
}
